docs/docs-app/components/header.js

```diff
@@ -1,44 +1,53 @@
 import React, {Component} from 'react';
 import {Link} from 'react-router';
 
-const tabs = [{
-  name: 'EXAMPLES',
-  path: '/examples'
-}, {
-  name: 'DOCUMENTATION',
-  path: '/documentation'
-}];
-
-const renderTab = (tab, index) => (
-  <Link
-    key={`${index}-link`}
-    className="link"
-    to={tab.path}
-    activeClassName="active">
-    {tab.name}
-  </Link>
-);
-
-const renderGithubLink = () => (
-  <a href="https://github.com/uber/react-vis" className="link" key="github-link">
-    <i className="icon icon-github" />
-  </a>
-);
-
-const mappedTabs = tabs.map(renderTab).concat(renderGithubLink());
+import {FRAMEWORK_NAME, FRAMEWORK_GITHUB_URL} from '../constants/framework';
+import FRAMEWORK_LINKS from '../constants/links';
 
 export default class Header extends Component {
+  _renderLinks() {
+    const links = Object.keys(FRAMEWORK_LINKS).filter(name => name !== FRAMEWORK_NAME);
+    return (
+      <div className="site-links">
+        <div className="site-link" key={FRAMEWORK_NAME}>
+          <a href="#">{FRAMEWORK_NAME}</a>
+        </div>
+        {
+          links.map(name =>
+            <div className="site-link" key={name}>
+              <a href={FRAMEWORK_LINKS[name]}>{name}</a>
+            </div>
+          )
+        }
+      </div>
+    );
+  }
 
   render() {
+    const {isMenuOpen, opacity, toggleMenu} = this.props;
+
     return (
-      <div className="example-header">
-        <Link to="/" className="link">
-          {'REACT VIS'}
-        </Link>
-        <div className="tabs-wrapper">
-          {mappedTabs}
+      <header className={isMenuOpen ? 'open' : ''}>
+        <div className="bg" style={{opacity}} />
+        <div className="container">
+          <a className="logo" href="#">
+            {FRAMEWORK_NAME}
```

Comment:
not sure abstracting out the name of the framework is all that useful unless other folks are using this same component, if thats the case then this shouldn't be copy and pasted in, it should be held in a common location